app/lib/put-file.js

```diff
@@ -1,3 +1,20 @@
+import createDOMPurify from 'dompurify';
+
+const DOMPurify = createDOMPurify(window);
+const sanitiseOptions = {
+  USE_PROFILES: {
+    svg: true,
+    svgFilters: true
+  },
+  ADD_TAGS: [
+    'animate',
+    'use'
+  ],
+  ADD_ATTR: [
+    'calcmode'
+  ]
```

Comment:
We could consider putting these behind a flag, so that project teams have to get our approval before they can upload unsafe code. And volunteers can't upload unsafe code at all.